net-analyzer/smbmap/smbmap-1.1.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit python-single-r1
```

Comment:
`python-single-r1` can be replaced to `python-r1`

Reply:
oh...

it seems this tool can write data files to a random directory
https://github.com/ShawnDEvans/smbmap/blob/b667382f1bb12359a63ecf8f6374b417ef2ae2fc/smbmap.py#L36-L50

maybe we need try to change it like this:
```diff
From 78b534e314590a1ed331393deaab9ef3aaeed07f Mon Sep 17 00:00:00 2001
From: Yury Martynov <email@linxon.ru>
Date: Fri, 1 Nov 2019 20:39:00 +0300
Subject: [PATCH] change workdir

---
 smbmap.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/smbmap.py b/smbmap.py
index 4db067c..a7792b6 100755
--- a/smbmap.py
+++ b/smbmap.py
@@ -33,11 +33,12 @@ import re
 # Seriously, the most amazing Python library ever!!
 # Many thanks to that dev team
 
-OUTPUT_FILENAME = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz',10))
-BATCH_FILENAME  = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 10)) + '.bat'
-SMBSERVER_DIR   = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 10))
+CURRENT_PATH = str(os.getcwd())
+OUTPUT_FILENAME = ''.join(CURRENT_PATH)
+BATCH_FILENAME  = ''.join(CURRENT_PATH) + '.bat'
+SMBSERVER_DIR   = ''.join(CURRENT_PATH)
 DUMMY_SHARE     = 'TMP'
-PERM_DIR = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 10))
+PERM_DIR = ''.join(CURRENT_PATH)
 
 class SMBServer(Thread):
     def __init__(self):
-- 
2.23.0


```

Reply:
Can't accept the patch. That would pollute the fs or fail to write a log if the current dir is read only.
It should write all output to ~/.smbmap or similar